File: Day2/quiz1/qn2.c
#include <stdio.h>
int main()
{
    int n;
    printf("Enter size of the array:");
    scanf("%d",&n);
    int arr[n];
    for(int i=0;i<n;i++)
    {
        printf("Enter element %d : ",i+1);
        scanf("%d",&arr[i]);
    }
    int temp=1;
    for(int i=0;i<n;i++)
    {
        if(arr[i]==arr[i+1])
        {
            temp++;
        }
        else
        {
            if(temp>1)
            {
                printf("%d -> %d\n",arr[i],temp);
            }
            temp=1;
        }
    }
    return 1;

}

File: Day6/quiz2/qn2.java
import java.util.Arrays;
import java.util.Scanner;

public class qn2 {
    public static void main(String[] args) {
        String str, ans = "";
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter String:");
        str = sc.nextLine();
        char[] arr = str.toCharArray();
        int count = 1;
        char temp = arr[0];
        for (int i = 1; i < arr.length; i++) {
            if (arr[i] == temp) {
                count++;
            } else {
                ans = ans + Character.toString(temp);
                ans = ans + Integer.toString(count);
                temp = arr[i];
                count = 1;
            }
        }
        ans = ans + Character.toString(temp);
        ans = ans + Integer.toString(count);
        System.out.println(ans);
    }
}